### ISSUE TITLE
Apply encoder/base model before lstm

### ISSUE THREAD

**Issue by remiresnap** [keras-team/keras#8375]
I have 10 images that I want to encode with a pre-trained encoder network (to 1024 features), and then apply a sequence-to-sequence lstm on the 10x1024 data.

Here is an example of a toy problem, without the encoder. The network takes as input (?, 10, 3).

    from numpy import array

    from keras.models import Model
    from keras.layers import LSTM, Dense, Bidirectional, Dropout, Input, GlobalAveragePooling2D

    from random import random

    def get_single_min_max_sequence(n_steps, n_features):
        x = [ [random() for _ in range(n_features)] for _ in range(n_steps) ]
        x_sum = [ sum(t) for t in x ]
        y = [ 0 for _ in range(n_steps)]
        y[x_sum.index(min(x_sum))] = 1
        y[x_sum.index(max(x_sum))] = 1
        y = [ [elem] for elem in y ]
        return x, y

    def gen_min_max_sequences(n_steps, n_features, n=64):
        while True:
            X, Y = [], []
            for _ in range(n):
                x, y = get_single_min_max_sequence(n_steps, n_features)
                X.append(x)
                Y.append(y)
            yield array(X), array(Y)

    def get_model(n_steps, n_features):
        inpt = Input(shape=(n_steps, n_features))

        x = Dense(128, activation="relu")(inpt)
        x = Dropout(0.5)(x)

        x = Bidirectional(LSTM(128, return_sequences=True, recurrent_dropout=0.5))(x)

        x = Dense(128, activation="relu")(x)
        x = Dropout(0.5)(x)

        x = Dense(1, activation="sigmoid", name="Output")(x)

        model = Model(inputs=inpt, outputs=x)
        model.compile(loss='binary_crossentropy', optimizer='adam', metrics=["binary_accuracy"])
        return model

    n_steps = 10
    n_features = 3
    print("Creating model...")
    model = get_model(n_steps, n_features)
    model.summary()

    print("Training model...")
    model.fit_generator(gen_min_max_sequences(n_steps, n_features), steps_per_epoch=128, epochs=128)

Now I would like to change get_model() so that it takes images (instead of 3d-vectors as in example). The input shape would be (?, 10, 224, 224, 3) for 10 images.
The encoder network could be the following:

    from keras.applications import MobileNet
    base_model = MobileNet(weights='imagenet', input_shape=(224, 224, 3), include_top=False)

I think the first few lines of the changed get_model() might be as follows:

    def get_model(base_model, n_images=10):
        base_model_output = GlobalAveragePooling2D()(base_model.output)
        print("base_model_output:", base_model_output.shape) # (?, 1024)
        inpt = Input(shape=(n_images, *base_model.input.shape[1:]))
        print("Input:", inpt.shape) # (?, 10, 224, 224, 3)

        # Combine inpt, base_model_output and Dense(128, activation="relu") ?
     
I am having trouble with correctly applying the base_model to all 10 input images, before feeding the encoded images to a dense layer (same as above) and then to the lstm. The weights in the base_model will be non-trainable.
